controllers/nonadmin_controller.go

```diff
@@ -190,10 +204,8 @@ func ensureRequiredSpecs(deploymentObject *appsv1.Deployment, image string, imag
 }
 
 func (r *DPAReconciler) checkNonAdminEnabled() bool {
-	dpa := r.dpa
-	if dpa.Spec.NonAdmin != nil &&
-		dpa.Spec.NonAdmin.Enable != nil {
-		return *dpa.Spec.NonAdmin.Enable
+	if r.dpa.Spec.NonAdmin != nil && r.dpa.Spec.NonAdmin.Enable != nil {
+		return *r.dpa.Spec.NonAdmin.Enable && r.dpa.Spec.UnsupportedOverrides[oadpv1alpha1.TechPreviewAck] == TrueVal
```

Comment:
Do we need to ensure the comparison is against "True", "TRUE" or "true", so regardless of capital/small letters?

```go
import (
	"strings"
[...]

		return *r.dpa.Spec.NonAdmin.Enable && strings.EqualFold(r.dpa.Spec.UnsupportedOverrides[oadpv1alpha1.TechPreviewAck], TrueVal)
```

Reply:
Ok thanks, comment can be closed.